cdptools/sr_models/webvtt_sr_model.py

```diff
@@ -4,15 +4,15 @@
 import io
 import json
 import logging
-import nltk
 import re
+from pathlib import Path
+from typing import Dict, List, Union
+
+import nltk
```

Comment:
Editor auto sort imports :woman_shrugging:

Reply:
Just for my knowledge, is there a pattern to structuring/ordering the imports?

Reply:
Yep. Details [here](https://www.python.org/dev/peps/pep-0008/#imports).

```
Standard library imports.
Related third party imports.
Local application/library specific imports.
```